PayPal/src/main/java/com/braintreepayments/api/paypal/vaultedit/PayPalEditRequestParams.kt

```diff
@@ -0,0 +1,12 @@
+package com.braintreepayments.api.paypal.vaultedit
+
+import androidx.annotation.RestrictTo
+import com.braintreepayments.api.BrowserSwitchOptions
+
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+data class PayPalVaultEditAuthRequestParams @JvmOverloads internal constructor(
```

Comment:
```suggestion
data class PayPalVaultEditAuthRequestParams internal constructor(
```

Do we need the `@JvmOverloads` annotation? I think all classes should already be converted to Kotlin.